keps/sig-apps/2599-minreadyseconds-for-statefulsets/kep.yaml

```diff
@@ -7,24 +7,25 @@ participating-sigs:
   - "sig-apps"
 status: implementable
 creation-date: 2021-04-07
-last-updated: 2021-04-12
+last-updated: 2022-06-08
 reviewers:
   - "@soltysh"
 approvers:
   - "@soltysh"
 prr-approvers:
   - "@ehashman"
+  - "@deads2k"
```

Comment:
wojtek-t per above change

Reply:
I don't care about it - it's a duplicate with the file in prod-readiness dir.
I'm planning to get to it at some point and remove this field from everywhere..